test/recipes/80-test_ssl_new.t

```diff
@@ -56,6 +56,7 @@ my $no_ocsp = disabled("ocsp");
 my %conf_dependent_tests = (
   "02-protocol-version.conf" => !$is_default_tls,
   "04-client_auth.conf" => !$is_default_tls,
+  "05-sni.conf" => $no_tls || disabled("tls1_1"),
```

Comment:
I don't understand why "no_tls" is required here.  As can be seen from the comment a few lines below this, the whole test is supposed to be skipped in the event of no_tls.